packages/cli/src/commands/account/new.ts

```diff
@@ -24,6 +24,18 @@ export default class NewAccount extends BaseCommand {
 
   static flags = {
     ...BaseCommand.flags,
+    privateKey: {
+      ...BaseCommand.flags.privateKey,
+      hidden: true,
```

Comment:
it would make no sense to use to use these flags on this command so keep them hidden like before

Reply:
hmmm I could see a world where you just wanna quickly see your account info (pubkey, address) from your private key?

Reply:
that may be true but adding this here just keeps the existing behavior 

in `celocli account:new --privateKey` 

we would have to create a new command or flag for what you are suggesting